### DISCUSSION TITLE
Transit network only on railway stations

### DISCUSSION THREAD

**Discussion by Oivo35** [osmlab/name-suggestion-index#10267] [Q&A] [answered]
Is it possible to have a preset only for railway stations, since the operator operates only the railway stations, but no train routes?

**Comment by UKChris-osm** [comment, marked as answer]
Yes, you can add a preset directly to the [/railway/station.json](https://github.com/osmlab/name-suggestion-index/blob/main/data/transit/railway/station.json) file, underneath the template code, similar to how it's done in the [/railway/tram_stop.json](https://github.com/osmlab/name-suggestion-index/blob/main/data/transit/railway/tram_stop.json) file.
